backend/src/pipeline.py

```diff
@@ -25,7 +25,7 @@
 from spectral_reflectance import spectrally_transform
 from verification import verify_color_transformation, verify_spectral_transformation
 
-def processing_pipeline(packet):
+def processing_pipeline(packet, outpath, colorspace):
     """ Color calibration pipeline
```

Comment:
I'll update comments before merging, I feel like this change shouldn't be controversial